lib/phobos/actions/process_message_batch.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module Phobos
+  module Actions
+    class BatchMessage
```

Comment:
Should preferably be in its own file called `batch_message.rb` placed in `lib/phobos`, it is much easier to find and write isolated tests etc for it.